validator/accounts/v2/wallet.go

```diff
@@ -246,6 +250,20 @@ func (w *Wallet) WriteFileAtPath(ctx context.Context, filePath string, fileName
 	return nil
 }
 
+// ReadFileAtPath within the wallet directory given the desired path and filename.
+func (w *Wallet) ReadFileAtPath(ctx context.Context, filePath string, fileName string) ([]byte, error) {
```

Comment:
There are too many ways to read and writes files in the wallet. In a separate PR, all of these will be simplified and deduplicated.